spec/my_enumerables_spec.rb

```diff
@@ -0,0 +1,226 @@
+require './my_enumerables.rb'
+
+describe Enumerable do
+  context '#my_each' do
```

Comment:
You use the same array `[1,2,3]` in many of the test cases. [Use let instead](http://www.betterspecs.org/#let)

Reply:
Same with repeated blocks, you can store it with `let` and use it when necessary

Reply:
Please refer to Commit No. #6. Alvaro, could you please advise on storing a block within a let. Done some research on that and have tried to no avail.

Reply:
A proc within a let where possible with latest commit see below